### ISSUE TITLE
Investigate polyfill path with Babel

### ISSUE THREAD

**Issue by guybedford** [jspm/jspm-cli#615]
Apparently `Object.assign` isn't working.

**Comment by guybedford** [comment]
This is working fine.

**Comment by LeDominik** [comment]
After upgrading to the brand-new 0.15.2 with the new babel 5 just done by @crisptrutski it happened again. This time I checked with multiple browsers and isolated the thing, I sincerely hope it's reproducible now (don't want to waste anyone's time). What I did:

1. Create new empty project, `jspm init` all defaults, except for babel instead of traceur
2. Minimal HTML
3. Minimal JavaScript using `Object.assign()`, forcing Babel

HTML is as easy as it get's:
```html
<!DOCTYPE html>
<html>
<head lang="en">
    <meta charset="UTF-8">
    <title>Assign test</title>
</head>
<body>
    <h1>Assign test</h1>
    <script src="jspm_packages/system.js"></script>
    <script src="config.js"></script>

    <script>
        System.import('lib/app');
    </script>
</body>
</html>
```

The lib/app.js is:
```javascript
let hello = {here: 'hereeee'};
Object.assign(hello, {we: 'go'});
console.log(hello);

// force babel transform, without this code would be passed through:
export default function () { };
```

Thanks to forcing babel the call to `Object.assign` is correctly transformed to `_core.Object.assign(hello, { we: 'go' });` and then I get this funky error as it's erroneously loading a non-existent `library.js` to satisfy the babel runtime:
```
Potentially unhandled rejection [3] Error loading "core-js/library" at http://127.0.0.1:8080/core-js/library.js
Error loading "core-js/library" from "lib/app" at http://127.0.0.1:8080/lib/app.js
```

I guess it's resulting out of the transformed JavaScript's dependencies (sourcemap stripped):
```javascript
(function() { var __moduleName = "lib/app"; var __moduleAddress = "http://127.0.0.1:8080/lib/app.js";System.register(['babel-runtime/core-js'], function (_export) {
  var _core, hello;

  return {
    setters: [function (_babelRuntimeCoreJs) {
      _core = _babelRuntimeCoreJs['default'];
    }],
    execute: function () {
      'use strict';

      hello = { here: 'hereeee' };

      _core.Object.assign(hello, { we: 'go' });
      console.log(hello);

      // force babel transform:
      _export('default', function () {});

      ;
    }
  };
});
 }).call(__global);
```

Any help appreciated!

**Comment by guybedford** [comment]
Thanks so so much - I've included this in the 0.15.3 release.

**Comment by LeDominik** [comment]
Wow, thanks -- works (of course) :+1:
